packages/mattermost-redux/src/types/websocket.ts

```diff
@@ -1,4 +1,22 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-export * from '@mattermost/types/websocket';
+export type WebSocketBroadcast = {
```

Comment:
This might belong in the types or client package again once we ship the websocket client